### DISCUSSION TITLE
Upgrading from 1.0.0-rc.1 to 1.0.0-rc.5

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#292] [Q&A] [answered]
hi
We are currently using 1.0.0-rc.1. Can we simply update to 1.0.0-rc.5 or should we do some special preparation, consider something ?

thanks

**Comment by fgatti675** [comment, marked as answer]
Hi @Andre3424,
yes, it should be no problem updating to the latest version since there are no breaking changes
Best,

**Reply by Amtech281** [reply]
thanks for the fast answer!
